### PR TITLE
RDM-9717 Fix CVE-2020-7637 on Case Man Web

### PR DESCRIPTION
**Pull request by skandala-cgi**
### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/RDM-9717

### Change description ###
Fix CVE-2020-7637 on Case Man Web

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[X] No
```

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -137,7 +136,8 @@
     "tslint": "^5.0.0",
     "typescript": "3.1.1",
     "webdriver-manager": "^12.1.0",
-    "webpack-cli": "^2.0.14"
+    "webpack-cli": "^2.0.14",
+    "class-transformer": "^0.3.1"
```
Comment (MSancaktutar) [inline package.json:140]: use 0.2.3 pls.
0.3.1 will fail IE 11.

**Review by MSancaktutar — APPROVED** [review]
lgtm

**Review by MSancaktutar — COMMENTED** [review]
@skandala-cgi  - Is this change also necessitating a yarn lock update?

**Comment by skandala-cgi** [comment]
> @skandala-cgi - Is this change also necessitating a yarn lock update?

No Mutlu. It is not updating yarn.lock file

**Review by MSancaktutar — APPROVED** [review]

**Pull request merged**